NeuralGraph/nfp.py

```diff
@@ -1,6 +1,7 @@
 from .model import QSAR
-import torch
+import torch as T
 from torch.utils.model_zoo import load_url
+dev = T.device('cuda') if T.cuda.is_available() else T.device('cpu')
```

Comment:
I think this line is redundant. 
can you try
`from .utils import dev` ?

Reply:
You are right. It works! :) Just submitted a new pr